### ISSUE TITLE
Documentation is not updated to the last released version

### ISSUE THREAD

**Issue by dalito** [devpi/devpi#927]
The linked documentation (https://doc.devpi.net/) is still for devpi-server-6.0.0 and devpi-client 5.2.2 (2021-02-19).

**Comment by fschulze** [comment]
Yes, that is a known issue. Generating the docs for various reasons is cumbersome and not automated and it didn't get much love in general. I hope to work on it soon.

**Comment by fschulze** [comment]
I worked on the automation and uploaded current docs. The docs themself need updates, but those are tracked in other tickets.
